scrapers/Vipissy.yml

```diff
@@ -0,0 +1,31 @@
+name: vipissy
+sceneByURL:
+  - action: scrapeXPath
+    url: 
+      - vipissy.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $performer: //dl/dd/a
+    scene:
+      Title: 
+        selector: //section[@class="downloads"]/strong/text()
+        replace:
+          - regex: ^(.*)[—]\s+
+            with:
+      Date:
+        selector: //dl/dd[2]/text()
+        parseDate: Jan 2,2006

```

Comment:
missed a space after the comma  `parseDate: Jan 2, 2006` -> `parseDate: Jan 2, 2006`
as stated for the other scraper  imo ` selector: //dl/dd[contains(txt(),"Released on:")]/text()`  is safer than `dd[2]`